packages/react/src/components/TextInput/TextInput.js

```diff
@@ -241,6 +238,11 @@ TextInput.propTypes = {
    */
   placeholder: PropTypes.string,
 
+  /**
+   * Whether the input should be read-only
+   */
+  readonly: PropTypes.string,
```

Comment:
```suggestion
  readonly: PropTypes.bool,
```

![image](https://user-images.githubusercontent.com/15822070/120365646-53d11a00-c2d4-11eb-956d-0a65aeebbece.png)

☝️ When the PropType is set to string, I get the following in my console. When I turn it into a bool it goes away

Reply:
Thanks for catching that!